### PR TITLE
Fix code samples in the README.md

### PR DESCRIPTION
**Pull request by Biruntha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: sfdc37/Module.md [hunk 1]
```diff
@@ -67,50 +67,56 @@ sfdc37:SalesforceConfiguration salesforceConfig = {
...
+
+The `createAccount` function creates an Account SObject. Pass a JSON object with the relevant fields needed for the SObject Account.
+
 ```ballerina
 json account = {Name:"ABC Inc", BillingCity:"New York"};
-var createReponse = salesforceEP->createAccount(account);
+var createReponse = salesforceClient->createAccount(account);
 ```
+
 The response from `createAccount` is either a string ID of the created account (if the account was created successfully)
```
Comment (MaryamZi) [inline sfdc37/Module.md]:
```suggestion
The response from `createAccount` is either the string ID of the created account (if the account was created successfully) 
```

File: sfdc37/Module.md [hunk 2]
```diff
@@ -67,50 +67,56 @@ sfdc37:SalesforceConfiguration salesforceConfig = {
 
 sfdc37:Client salesforceClient = new(salesforceConfig);
 ```
-The `createAccount` function creates an Account SObject. Pass a JSON object with the relevant fields needed for the 
-SObject Account.
+
+The `createAccount` function creates an Account SObject. Pass a JSON object with the relevant fields needed for the SObject Account.
```
Comment (MaryamZi) [inline sfdc37/Module.md]:
```suggestion
The `createAccount` remote function creates an Account SObject. Pass a JSON object with the relevant fields needed for the SObject Account.
```

File: sfdc37/Module.md [hunk 3]
```diff
@@ -67,50 +67,56 @@ sfdc37:SalesforceConfiguration salesforceConfig = {
...
 ```ballerina
 if (createReponse is string) {
     io:println("Account id:  " + createReponse);
 } else {
     io:println(createReponse.message);
 }
 ```
-The `getQueryResult` function executes a SOQL query that returns all the results in a single response, or, if it exceeds 
+
+The `getQueryResult` function executes a SOQL query that returns all the results in a single response or if it exceeds
```
Comment (MaryamZi) [inline sfdc37/Module.md]:
```suggestion
The `getQueryResult` remote function executes a SOQL query that returns all the results in a single response or if it exceeds
```

File: sfdc37/Module.md [hunk 4]
```diff
@@ -67,50 +67,56 @@ sfdc37:SalesforceConfiguration salesforceConfig = {
...
 the maximum record limit, it returns part of the results and an identifier used to retrieve the remaining results.
```
Comment (MaryamZi) [inline sfdc37/Module.md]:
```suggestion
the maximum record limit, it returns part of the results and an identifier that can be used to retrieve the remaining results.
```

File: sfdc37/Module.md [hunk 5]
```diff
@@ -67,50 +67,56 @@ sfdc37:SalesforceConfiguration salesforceConfig = {
...
 ```ballerina
 if (response is json) {
-    io:println(“TotalSize:  ” + response["totalSize"]);
-    io:println(“Done:  ” + response["done"]);
-    io:println(“Records:  ” + response["records"]);
-    io:println(“Next response url:”  + response["nextRecordsUrl"]);
+    io:println("TotalSize:  ", response["totalSize"]);
+    io:println("Done:  ", response["done"]);
+    io:println("Records: ", response["records"]);
+    io:println();
```
Comment (MaryamZi) [inline sfdc37/Module.md]: Do we need this?

File: README.md [hunk 6]
```diff
@@ -63,23 +63,25 @@ string refreshToken = "<refresh_token>";
...
-    };
+    }
+};
 
-    sfdc37:Client salesforceClient = new(salesforceConfig);
+sfdc37:Client salesforceClient = new(salesforceConfig);
+
+public function main() {
 
     //Call the Salesforce connector function getAvailableApiVersions().
```
Comment (Shan1024) [inline README.md]:
```suggestion
    // Call the Salesforce connector function getAvailableApiVersions().
```

File: README.md [hunk 7]
```diff
@@ -63,25 +63,27 @@ string refreshToken = "<refresh_token>";
...
+    }
+};
 
-    sfdc37:Client salesforceClient = new(salesforceConfig);
+sfdc37:Client salesforceClient = new(salesforceConfig);
+
+public function main() {
 
-    //Call the Salesforce connector function getAvailableApiVersions().
+    // Call the Salesforce connector function getAvailableApiVersions().
```
Comment (MaryamZi) [inline README.md]:
```suggestion
    // Call the `getAvailableApiVersions()` remote function of the Salesforce connector.
```

File: README.md [hunk 8]
```diff
@@ -63,33 +63,35 @@ string refreshToken = "<refresh_token>";
...
+public function main() {
 
-    //Call the Salesforce connector function getAvailableApiVersions().
+    // Call the `getAvailableApiVersions()` remote function of the Salesforce connector.
     json|sfdc37:SalesforceConnectorError response = salesforceClient->getAvailableApiVersions();
 
     if (response is json) {
-        //if successful, returns JSON result
-        io:println(response);
+        // If successful, print JSON result
```
Comment (MaryamZi) [inline README.md]:
```suggestion
        // If successful, print the JSON result
```

File: README.md [hunk 9]
```diff
@@ -63,33 +63,35 @@ string refreshToken = "<refresh_token>";
...
+        io:println("Available Api versions: ", response);
```
Comment (MaryamZi) [inline README.md]:
```suggestion
        io:println("Available API versions: ", response);
```

File: README.md [hunk 10]
```diff
@@ -63,33 +63,35 @@ string refreshToken = "<refresh_token>";
...
     } else {
-        //if unsuccessful, returns an error of type sfdc37:SalesforceConnectorError
-        io:println(response.message);
+        // If unsuccessful, print an error of type sfdc37:SalesforceConnectorError
```
Comment (MaryamZi) [inline README.md]:
```suggestion
        // If unsuccessful, print the error of type `sfdc37:SalesforceConnectorError`
```

File: README.md [hunk 11]
```diff
@@ -63,33 +63,35 @@ string refreshToken = "<refresh_token>";
...
+        io:println("Error: ", response.message);
```
Comment (MaryamZi) [inline README.md]:
```suggestion
        io:println("Error: ", response);
```

**Pull request merged**